frontend/src/constants.js

```diff
@@ -0,0 +1,224 @@
+const serverURL = "http://localhost:8080";
+
+const emptyGameStateSchema = {
+  players: {},
+  playerOrder: [],
+  currentPlayer: "",
+  ownedPowerPlants: {},
+  ownedResources: {},
+  ownedCities: {},
+  actualMarket: [],
+  futureMarket: [],
+  gamePhase: "",
+  currentBid: 0,
+  highestBidder: ""
+};
+
+const emptyClientGameStateSchema = {
+  userProfile: {
+    name: "",
+    color: "",
+    uuid: ""
+  },
+  gameState: emptyGameStateSchema,
+  showGame: false,
+  showPowerPlantMarket: false,
+  highestBidder: "",
+  selectedCities: [],
+  costToPurchaseCities: 0
+};
+
+const translationsNeeded = [
```

Comment:
Subjective: This is neat! Although it does smell a bit like eager over-abstraction. Might have been better to do a more direct inline translation.